libraryBuilds/com.fasterxml.jackson.core/com.fasterxml.jackson.core-jackson-databind/src/test/java/com/github/j2objccontrib/j2objcgradle/failing/GenericsTest.java

```diff
@@ -0,0 +1,193 @@
+package com.github.j2objccontrib.j2objcgradle.failing;
```

Comment:
As you've said you wrote this, please add the standard project copyright notice (see other files)